client/src/components/players/OmniPlayer.vue

```diff
@@ -409,4 +437,17 @@ const renderedSpans = computed(() => {
 	font-size: 12px;
 	z-index: 500;
 }
+.playback-error {
+	position: absolute;
+	width: 100%;
+	height: 100%;
+	align-content: center;
+	background-color: rgba(var(--v-theme-surface), 1);
```

Comment:
I think its more consistent if we use the background color, not the surface color